jax/_src/numpy/ufuncs.py

```diff
@@ -374,9 +374,47 @@ def exp(x: ArrayLike, /) -> Array:
   """
   return lax.exp(*promote_args_inexact('exp', x))
 
-@implements(np.log, module='numpy')
+
 @partial(jit, inline=True)
 def log(x: ArrayLike, /) -> Array:
+  """Calculate element-wise natural logarithm of the input.
+
+  JAX implementation of :obj:`numpy.log`.
+
+  Args:
+    x: input array or scalar.
+
+  Returns:
+    An array containing the logarithm of each element in ``x``, promotes to inexact
+    dtype.
+
+  See also:
+    - :func:`jax.numpy.exp`: Calculates element-wise exponential of the input.
+    - :func:`jax.numpy.log2`: Calculates base-2 logarithm of each element of input.
+    - :func:`jax.numpy.log1p`: Calculates element-wise logarithm of one plus input.
+
+  Examples:
+    ``jnp.log`` and ``jnp.exp`` are inverse functions of each other. Applying
+    ``jnp.log`` on the result of ``jnp.exp(x)`` yeilds the original input ``x``.
+
+    >>> x = jnp.array([2, 3, 4, 5])
+    >>> jnp.log(jnp.exp(x))
+    Array([2., 3., 4., 5.], dtype=float32)
+    >>> jnp.allclose(x, jnp.log(jnp.exp(x)))
+    Array(True, dtype=bool)
+
+    ``jnp.log`` follows the properties of exponential such as :math:`log(a*b) =
```

Comment:
This seems like strange phrasing. Why would `jnp.log` not follow the properties of logarithms? How about something like this: "Using ``jnp.log`` we can demonstrate well-known properties of logarithms, such as..."